hypex/ab_test/ab_tester.py

```diff
@@ -745,9 +743,9 @@ def process(self, data: pd.DataFrame, optimize_groups: bool = False, iterations:
 
```

Comment:
line 735 
what is the usage of variable aa_scores? 
ou don't use it after initialization